docs/pages/api-reference/connectors/pubsub.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Pub/Sub
+order: 0
+status: published
+---
+### Pub/Sub
+Data connector to Google Pub/Sub messaging service.
+
+#### Project Parameters
+<Expandable title="name" type="str">
+A name to identify the source. The name should be unique across all Fennel sources.
+</Expandable>
+
+<Expandable title="project_id" type="str">
+The project ID of the Google Cloud project containing the Pub/Sub topic
+</Expandable>
+
+<Expandable title="service_account_key" type="str">
+The JSON string containing the credentials for the Service Account to use to access
+Pub/Sub topic. See below for instructions on how to obtain this.
```

Comment:
This isn't a json string but a dictionary, right? If so, please change this line to mention that.

Reply:
I copied this from BigQuery and forgot to update. Updated now